.github/workflows/ci.yml

```diff
@@ -28,6 +28,8 @@ jobs:
         include:
           - { os: windows-latest , ruby: mingw }
           - { os: windows-latest , ruby: mswin }
+          - { os: ubuntu-latest , ruby: truffleruby }
+          - { os: ubuntu-latest , ruby: jruby }
```

Comment:
Why do you want to reuse this job?
It seems that you just want to test `rake install` for TruffleRuby and JRuby.

Reply:
My idea is to do that to avoid duplicating lines 38 to 60, to ensure they stay in sync.
And it also makes it easier to later enable tests e.g. for TruffleRuby once all or almost all Fiddle tests pass.

Given that, is it OK as-is or you prefer a separate job with some duplication?

Reply:
Do we need to duplicate lines 38 to 60?

It seems that we just need this:

```yaml
      - uses: actions/checkout@v4
      - name: Set up Ruby
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: ${{ matrix.ruby }}
          bundler-cache: true
      - run: bundle exec rake install
      - run: ruby -e 'require "fiddle"'
```

> And it also makes it easier to later enable tests e.g. for TruffleRuby once all or almost all Fiddle tests pass.

It there a plan to work on this? If there isn't a plan for this, we don't need to care about this.

> Given that, is it OK as-is or you prefer a separate job with some duplication?

Could you create a separate job? We don't want to maintain `if:` as much as possible.

Reply:
Alright, let's use a separate job. I'm unsure if we need the version bump as well.

Reply:
Done